app/init/global_event_handler.ts

```diff
@@ -141,7 +144,9 @@ class GlobalEventHandler {
         this.configureAnalytics(serverUrl, config);
 
         // TODO: Add minimum server version checked with cloud support
-        // if (isMinimumServerVersion(Client4.serverVersion, 5, 24) && config.ExtendSessionLengthWithActivity === 'true') {
+        // TODO: get serverVersion from database
+        // const serverVersion = NetworkManager.clients[serverUrl].serverVersion;
+        // if (isMinimumServerVersion(serverVersion, 5, 24) && config.ExtendSessionLengthWithActivity === 'true') {
```

Comment:
5.24 is no longer supported, server Version should not be a thing here, just check the config value